docs/config.toml

```diff
@@ -37,3 +37,20 @@ publishDir = "../out/docs/"
 [[menu.extra]]
     name = "submit an issue"
     url = "https://github.com/workspacer/workspacer/issues/new"
+
+[[menu.footer]]
+    name = "Workspacer"
+    url = "https://github.com/workspacer/workspacer"
+    weight = 1
+    
+[[menu.footer]]
```

Comment:
Since workspacer is now a community project, shouldn't we remove this?

Reply:
(line 46 - 49) not sure why it's displayed like this

Reply:
Maybe, but I think Rick deserves some credit for his work.